### PR TITLE
Add support for gojs

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
Got hello world working

# Review

- [ ] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

File: lib/go-js/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = "wasmer-go-js"
+version = "0.10.2"
+authors = ["The Wasmer Engineering Team <engineering@wasmer.io>"]
+edition = "2018"
+repository = "https://github.com/wasmerio/wasmer"
+# TOOO: change below when finished
+publish = false
+description = "High-Performance WebAssembly JIT interpreter"
```
Comment (Hywan) [inline lib/go-js/Cargo.toml]: The description should be updated.

**Comment by MarkMcCaskey** [comment]
> Is it ready for a review?

It is in the sense that it works, but I haven't cleaned it up or added doc comments yet (there's lots of dead code still around)

**Comment by maxmcd** [comment]
this is all very exciting to see :)

**Comment by derekparker** [comment]
Is there anything (aside from a review) blocking this from landing? I'd really like to run Go WASM binaries via wasmer-runtime embedded in a Rust program "natively" without having to define these syscalls myself.

**Comment by Hywan** [comment]
The Wasm modules compiled by the Go compiler are very tailored to JavaScript. It can also break at anytime. We have decided to not continue on that path for the moment. There is a similar PR in `wasmer-go`, https://github.com/wasmerio/wasmer-go/pull/95.

If you want to get Go compiled to Wasm with the WASI ABI, please use [TinyGo](https://tinygo.org/), and compile your Go programs as follows:

```shell
$ tinygo build -o ./module.wasm -target wasi .
```

That way, your resulting WebAssembly module with run with Wasmer, any many other runtimes.

**Pull request closed without merging**